infrastructure/staking-mechanics/offenses-and-slashes.md

```diff
@@ -1,4 +1,179 @@
+
+### Invalid Votes
+
+A validator will be penalized for inappropriate voting activity during the block inclusion and approval processes. The invalid voting related offenses are as follows:
+
+- **Backing an invalid block** - a para-validator backs an invalid block for inclusion in a fork of the relay chain
+- **`ForInvalid` vote** - when acting as a secondary checker, the validator votes in favor of an invalid block
+- **`AgainstValid` vote** - when acting as a secondary checker, the validator votes against a valid block. This type of vote wastes network resources required to resolve the disparate votes and resulting dispute
+
+### Equivocations
+
+Equivocation occurs when a validator produces statements that conflict with each other when producing blocks or voting. Unintentional equivocations usually occur when duplicate signing keys reside on the validator host. If keys are never duplicated, the probability of an honest equivocation slash decreases to near zero. The equivocation related offenses are as follows:
+
+- **Equivocation** - the validator produces two or more of the same block or vote
+    - **GRANDPA and BEEFY equivocation** - the validator signs two or more votes in the same round on different chains
+    - **BABE equivocation** - the validator produces two or more blocks on the relay chain in the same time slot
+- **Double seconded equivocation** - the validator attempts to second, or back, more than one block in the same round
+- **Seconded and valid equivocation** - the validator seconds, or backs, a block and then attempts to hide their role as the responsible backer by later placing a standard validation vote
+
+## Penalties
+
+On Polkadot, offenses to the network incur different penalties depending on severity. There are three main penalties: slashing, deactivation, and reputation changes.
+
+### Slashing
+
+Validators engaging in bad actor behavior in the network may be subject to slashing if they commit a qualifying offense. When a validator is slashed, they and their nominators lose a percentage of their staked DOT or KSM, from as little as 0.01% up to 100% based on the severity of the offense. Nominators are evaluated for slashing against their active validations at any given time. Validator nodes are evaluated as discrete entities, meaning an operator can't attempt to mitigate the offense on another node they operate in order to avoid a slash. 
+
+Any slashed DOT or KSM will be added to the [Treasury](https://wiki.polkadot.network/docs/learn-polkadot-opengov-treasury){target=\_blank} rather than burned or distributed as rewards. Moving slashed funds to the Treasury allows an opportunity to quickly move tokens away from malicious validators while maintaining the ability to revert faulty slashes when needed.
```

Comment:
```suggestion
Any slashed DOT or KSM will be added to the [Treasury](https://wiki.polkadot.network/docs/learn-polkadot-opengov-treasury){target=\_blank} rather than burned or distributed as rewards. Moving slashed funds to the Treasury allows tokens to be quickly moved away from malicious validators while maintaining the ability to revert faulty slashes when needed.
```